pkg/oc/cli/describe/describer.go

```diff
@@ -1202,6 +1203,93 @@ func (d *TemplateDescriber) DescribeTemplate(template *templateapi.Template) (st
 	})
 }
 
+// TemplateInstanceDescriber generates information about a template instance
+type TemplateInstanceDescriber struct {
+	kubeClient     kclientset.Interface
+	templateClient templateclient.TemplateInterface
+	kprinters.ObjectDescriber
+}
+
+// Describe returns the description of a template instance
+func (d *TemplateInstanceDescriber) Describe(namespace, name string, settings kprinters.DescriberSettings) (string, error) {
+	c := d.templateClient.TemplateInstances(namespace)
+	templateInstance, err := c.Get(name, metav1.GetOptions{})
+	if err != nil {
+		return "", err
+	}
+	return d.DescribeTemplateInstance(templateInstance, namespace, settings)
+}
+
+// DescribeTemplateInstance prints out information about the template instance
+func (d *TemplateInstanceDescriber) DescribeTemplateInstance(templateInstance *templateapi.TemplateInstance, namespace string, settings kprinters.DescriberSettings) (string, error) {
+	return tabbedString(func(out *tabwriter.Writer) error {
+		formatMeta(out, templateInstance.ObjectMeta)
+		out.Write([]byte("\n"))
```

Comment:
you could replace this with `fmt.Fprintln(out)`

Reply:
Was trying to keep in line with the way that they rest of the describers do it.